src/numerics/Newton.cpp

```diff
@@ -0,0 +1,290 @@
+//! @file Newton.cpp damped Newton solver
```

Comment:
Modified version of `MultiNewton` for single ODE systems, no longer depends on `OneDim` module. Planning to add time stepping capability to this class.